app.py

```diff
@@ -0,0 +1,112 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from dataclasses import dataclass
+from args import get_args
+import typing
+import requests
+import json
+import uvicorn
+import yaml
+import threading
+import time
+
+args = get_args()
+with open(args.config, "r") as stream:
+    try:
+        data = yaml.safe_load(stream)
+        API_KEY = data.get("api_key", None)
+        stops = data.get("stops", {})
+    except yaml.YAMLError as exc:
+        print(exc)
```

Comment:
how about if the yaml data isnt formatted correctly or the file isnt found? we should log and exit

lets make this open and yaml.safe_load its own function inside of `args.py`

Reply:
instead of print, we can use logging which is built into python

```py
import logging
import time


# Always log in UTC and output in RFC3339
logging.Formatter.converter = time.gmtime

formatter = logging.basicConfig(
    # https://docs.python.org/3/library/logging.html#logrecord-attributes
    format="%(asctime)s.%(msecs)03dZ %(threadName)s %(levelname)s:%(name)s:%(message)s",
    datefmt="%Y-%m-%dT%H:%M:%S",
    level=logging.INFO,
)
```

Reply:
```python
except Exception as error:
        logging.error(error)
        sys.exit(1)
```
Would this be okay? it's on [176b705](https://github.com/SCE-Development/investigate-511/pull/1/commits/176b705cd2bf8f8161e3c6b78046426999359e3a)

Reply:
done